api/_lib/parser.ts

```diff
@@ -35,8 +39,10 @@ export function parseRequest(req: IncomingMessage) {
     fileType: extension === 'jpeg' ? extension : 'png',
     text: decodeURIComponent(text),
     theme: theme === 'dark' ? 'dark' : 'light',
+    backgroundType: backgroundType === 'image' ? 'image' : 'pattern',
     md: md === '1' || md === 'true',
     fontFamily: fontFamily === 'roboto-condensed' ? 'Roboto Condensed' : 'Source Sans Pro',
+    imageUrl: imageUrl === ''? '': '',
```

Comment:
This logic doesn't make sense to me...it says "If the imageUrl is an empty string, return an empty string. Otherwise return an empty string".

Reply:
@zackkrida I agree with you completely on this, kindly refer to [this](https://github.com/creativecommons/og-image-generator/pull/20#discussion_r584025919) discussion once where I have gone in-depth with issues faced by me, and why I had to use the syntax that I used. I would love your inputs